src/azure/attestation/test/attestation_test.cc

```diff
@@ -134,4 +136,64 @@ TEST_F(JsonAttestationReportTest, FetchRealAttestationNonSnp) {
   }
   EXPECT_FALSE(fetchSnpAttestation().has_value());
 }
+
+TEST_F(JsonAttestationReportTest, GetReport) {
+  if (!hasSnp()) {
+    return;
+  }
+  // Define test inputs and outputs
+  std::string report_data = "";
+  uint8_t expected[64] = {
```

Comment:
Yes for these tests since the ask was about minimal transformations on the inputs or outputs I kept as uint8_t[64] but I'm very pro using std::array so I'll make that change

Reply:
Addressed in https://github.com/KenGordon/data-plane-shared-libraries/pull/49/commits/c747c834323474ad2c624c92470f40f59f729099